src/RoslynPad/MainWindow.xaml

```diff
@@ -144,11 +151,28 @@
                                    HorizontalAlignment="Center" />
                     </StackPanel>
                 </Button>
+
```

Comment:
This should be inside the docking manager where the tree view is. Wrap it in a dock panel and dock this stack panel to the bottom.